lib/hls/hls_parser.js

```diff
@@ -890,6 +890,25 @@ shaka.hls.HlsParser = class {
     /** @type {!Map<string, string>} */
     let mediaVariables = new Map();
 
+    this.manifest_ = {
```

Comment:
Why was this moved?  It's okay here, I just can't see why it changed.

Reply:
See https://github.com/shaka-project/shaka-player/pull/9506#discussion_r2623000747